tests/integration-tests-1.7.7.x/cypress.json

```diff
@@ -6,9 +6,9 @@
         "runMode": 2,
         "openMode": 0
     },
-    "pageLoadTimeout": 90000,
+    "pageLoadTimeout": 180000,
     "env": {
-      "runDiscountsTests": true,
+      "runPaymentsTests": false,
```

Comment:
Why is this default to false?

Reply:
@emicha Klarna made changes in the payment form which is causing the Klarna integration tests to fail Temporarily set to false until any solution is found to handle the form changes in the tests. After handling it, this default setting will be removed and we'll be running all tests.

And, this Klarna change is causing our all CMS integration tests to fail. I'll be creating tickets for all of the CMS to handle this change.

Reply:
I recommend you add a comment in the code itself, so it is easier to understand why this has been set to false.

Reply:
@emicha alright, adding the comment.

Reply:
Klarna payment form resolution has been found, created a ticket for this and added in the current sprint. That default setting will be updated after fixing. And, the fix will be pushed in this pull request.